### ISSUE TITLE
Find theme selling directories

### ISSUE THREAD

**Issue by Chimdiya1** [hng-teamfury-org/frontend#1]
We need to find places to place our themes up for download.
I was able to find the following:-

- [Themeforest](https://themeforest.net/)

- [Templatic](https://templatic.com/)

- [Flippa](https://www.flippa.com/)

I do not know if we're meant to choose one or if we can list on multiple sites(we should ask the mentors). Others should please drop their suggestions.

**Comment by Christie-code** [comment]
Theme forest is a good one
https://themeforest.net/search/sell

**Comment by fakoredeDamilola** [comment]
Please we have to pick one that is very popular. this task is about downloads

**Comment by Chimdiya1** [comment]
> Please we have to pick one that is very popular. this task is about downloads

that is why i posted the ones i found, the most popular one might not be even there.

**Comment by codessage** [comment]
`3. For frontend developers, you must recruit designers to your team who have previously designed hotel websites. You are to implement these hotel websites and upload to theme selling directories and make them free. The number of downloads you get across all is what counts to your points. As many designs as possible. But do not do what another team has done. (edited) `

it seems we can post across multiple platforms

**Comment by Chimdiya1** [comment]
> `3. For frontend developers, you must recruit designers to your team who have previously designed hotel websites. You are to implement these hotel websites and upload to theme selling directories and make them free. The number of downloads you get across all is what counts to your points. As many designs as possible. But do not do what another team has done. (edited) `
> 
> it seems we can post across multiple platforms

i guess these are okay and we should probably add more then

**Comment by emmanuelhashy** [comment]
> `3. For frontend developers, you must recruit designers to your team who have previously designed hotel websites. You are to implement these hotel websites and upload to theme selling directories and make them free. The number of downloads you get across all is what counts to your points. As many designs as possible. But do not do what another team has done. (edited) `
> 
> it seems we can post across multiple platforms

This is a nice idea. Yes we can.

**Comment by emmanuelhashy** [comment]
> We need to find places to place our themes up for download.
> I was able to find the following:-
> 
> * [Themeforest](https://themeforest.net/)
> * [Templatic](https://templatic.com/)
> * [Flippa](https://www.flippa.com/)
> 
> I do not know if we're meant to choose one or if we can list on multiple sites(we should ask the mentors). Others should please drop their suggestions.

Since there is no restrictions to the number of directories we can place our theme for download, we can upload our theme to as many theme selling directories available so that we can get a good number of downloads.

**Comment by MoronkeO** [comment]
Check out [Templatemonster ](templatemonster.com) too

**Comment by Chimdiya1** [comment]
> Check out [Templatemonster ](https://www.templatemonster.com/best-selling-website-templates/)  too

Nice, thanks. Others should suggest anyone they have also

**Comment by Marvellous1** [comment]
I think themeforest would be best because of its popularity

**Comment by costamay** [comment]
> I think themeforest would be best because of its popularity

I agree with you it is more popular and widely used, we need to keep the fire burning devs.

**Comment by Chimdiya1** [comment]
> I think themeforest would be best because of its popularity

We can use more that one

**Comment by olabode-dev** [comment]
Did some little research, I already checked out sites that we can upload our themes to but it will be done by HNG because, they will have to create an account, since they will be uploading via their platform. check the sites out
https://account.envato.com/, https://themeforest.net/cart and https://creativemarket.com/

**Comment by olabode-dev** [comment]
So we will just have to suggest the site, do our work, and leave the uploading to them

**Comment by adekniyi** [comment]
And we should do that before Thursday 
I think Mark stated that

**Comment by Technelly** [comment]
Theme forest is actually good. its popular
